### PR TITLE
feat: Add TextArea Feature

### PR DESCRIPTION
**Pull request by Defario**
Developer: @Defario

added Input Component : TextArea

TextArea.tsx, TextAreaProps.ts, styles.ts Done

### CODE REVIEW

File: src/components/Input/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { cx } from '@emotion/css';
+import React, { PropsWithChildren } from 'react';
+import { TextAreaProps } from './TextAreaProps';
+import { textAreaStyle, stySizeDefault, styStatusDefault } from './styles';
+
+const TextArea = (props: PropsWithChildren<TextAreaProps>): JSX.Element => {
```
Comment (randiadiel) [inline src/components/Input/TextArea/TextArea.tsx:6]:
```suggestion
const TextArea = (props: PropsWithChildren<TextAreaProps>) => {
```
sebenernya tuh typescript pinter enough to know that youre returning a JSX.Element..

unnecessary things clutters up the code, isn't it?

File: src/components/Input/TextArea/TextArea.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    const inputStatus =
+        txtAreaStatus !== undefined
+            ? `${txtAreaStatus} ${textAreaStyle}`
+            : cx(styStatusDefault);
+
+    return (
+        <div>
+            <textarea
+                className={`${inputSize} ${inputStatus}`}
```
Comment (randiadiel) [inline src/components/Input/TextArea/TextArea.tsx:22]:
```suggestion
                className={cx(inputSize, inputStatus)}
```
gini aja

File: src/components/Input/TextArea/TextArea.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    const inputSize =
+        txtAreaSize !== undefined
+            ? `${txtAreaSize} ${textAreaStyle}`
+            : cx(stySizeDefault);
+
+    const inputStatus =
+        txtAreaStatus !== undefined
+            ? `${txtAreaStatus} ${textAreaStyle}`
+            : cx(styStatusDefault);
```
Comment (randiadiel) [inline src/components/Input/TextArea/TextArea.tsx:17]: sebenernya ada cara yang lebih bagus, but for now no worries.. karena jelasinnya juga susah wkwk

File: src/components/Input/TextArea/TextArea.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { cx } from '@emotion/css';
+import React, { PropsWithChildren } from 'react';
+import { TextAreaProps } from './TextAreaProps';
+import { textAreaStyle, stySizeDefault, styStatusDefault } from './styles';
+
+const TextArea = (props: PropsWithChildren<TextAreaProps>): JSX.Element => {
+    const { placeholder, text, txtAreaSize, txtAreaStatus } = props;
```
Comment (randiadiel) [inline src/components/Input/TextArea/TextArea.tsx:7]: you should actually put default value here
```suggestion
    const { placeholder, text, txtAreaSize = 20, txtAreaStatus = 'status' } = props;
```

File: src/components/Input/index.tsx [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
     TextField: typeof TextField;
+    TextArea: typeof TextArea;
 }
 
 const Input = InternalInput as CompoundedComponent;
 
 Input.TextField = TextField;
+Input.TextArea = TextArea;
 
 export { default } from './Input';
```
Comment (randiadiel) [inline src/components/Input/index.tsx:18]: yarn.lock buat apa? buang aja changesnya itu.

Comment (Defario) [reply src/components/Input/index.tsx:18]: udah di synchronize sama master ko

**Pull request merged**